settings.gradle

```diff
@@ -1 +1,2 @@
 include ':Snapcast'
+include ':libs:libsnapcast'
```

Comment:
Gradle is complaining about this: `Unsupported Modules Detected: Compilation is not supported for following modules: libs, libsnapcast. Unfortunately you can't have non-Gradle Java modules and Android-Gradle modules in one project.`